src/components/DateTimePicker/Internal/ocpicker.module.scss

```diff
@@ -1459,7 +1459,7 @@
     }
 
     &-column {
-      @include scroll-bars();
+      @include scroll-bars;
```

Comment:
so if you don't have params you don't need the `()`?

Reply:
Nope. There was this inconsistency in the separate implementations, such that it made it clear these are not needed if no params are present.